ansible/configs/ocp4-cluster/default_vars.yml

```diff
@@ -54,19 +54,23 @@ install_ftl: false
 
 # TODO: Decide on whether to use sat or give access to repos directly with key
 # This will tell Agnosticd to use either:
-# sattelite, rhn, or file for repos
+# satellite, rhn, or file for repos
+# Only satellite is supported for RHEL 8
 repo_method: file
```

Comment:
If we're setting RHEL 8 as the default for these providers, we should switch this to Satellite at the default.

Reply:
I set satellite for ec2 and osp. Don't know if we have RHEL 8 images for GCP and Azure...

Reply:
I see them set in sample vars. They should be in default vars for those two cloud providers since RHEL 8 will not work with file repo_method at this time.

Reply:
OK I can restructure after.

Reply:
done